### ISSUE TITLE
Add parameter specifying the positive class in Precision metric

### ISSUE THREAD

**Issue by andrey-khropov** [catboost/catboost#2529]
catboost version: 1.2.2

[Scikit-learn's precision_score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_score.html) has `pos_label` parameter that specifies the class for calculate precision for.

@fingoldo [requested a similar parameter for CatBoost's Precision metric](https://github.com/catboost/catboost/issues/2422#issue-1785907829)
